### PR TITLE
Enhancement/houma pd cprr

### PR DESCRIPTION
**Pull request by ayyubibrahimi**

### CODE REVIEW

File: clean/houma_pd_cprr.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        .pipe(clean_ia_tracking_number)\
+        .pipe(extract_action)\
+        .pipe(clean_disposition)\
+        .pipe(split_and_clean_investigator)\
+        .pipe(split_and_clean_names)\
+        .pipe(drop_rows_missing_first_and_last_name)\
+        .pipe(standardize_desc_cols, ['tracking_number', 'action'])\
+        .pipe(set_values, ({'agency': 'Houma PD'}))\
+        .pipe(gen_uid, ['agency', 'first_name', 'last_name'])\
+        .pipe(gen_uid, ['uid', 'tracking_number', 'external_tracking_number', 'allegation', 'action'], 'complaint_uid')
```
Comment (pckhoi) [inline clean/houma_pd_cprr.py:129]: should be `allegation_uid` no?

Comment (ayyubibrahimi) [reply clean/houma_pd_cprr.py:129]: Yes. Thank you.

**Review by pckhoi — APPROVED** [review]
LGTM

**Comment by ayyubibrahimi** [comment]
I incorrectly merged here before pushing the changes because there are now an assortment of bugs in main. Please advise @pckhoi

**Comment by pckhoi** [comment]
What was the error that you saw?

**Comment by ayyubibrahimi** [comment]
Below is what I initially found:

in columns.py: the function `rearrange_complaint_columns` is again `rearrange_complaint_columns` despite recently being changed to `rearrange_allegation_columns`;

In clean: `baker_pd_cprr.py` is missing; 

In match: `baker_pd.py` is missing;

In raw_datasets.json: blocks for `baker_pd` and `houma_pd` are missing .

**Comment by pckhoi** [comment]
Email me a live share link, I could take a look quickly.

**Comment by ayyubibrahimi** [comment]
Just emailed it. The link is [here ](https://prod.liveshare.vsengsaas.visualstudio.com/join?21A8CBD01E06BE9927A871ABFEB478ECCFEA)as well.

**Pull request merged**